jenkins/databricks/setup.sh

```diff
@@ -54,8 +56,6 @@ fi
 # Set the path of python site-packages, and install packages here.
 PYTHON_SITE_PACKAGES="$HOME/.local/lib/python${PYTHON_VERSION}/site-packages"
 # Use "python -m pip install" to make sure pip matches with python.
-$PYSPARK_PYTHON -m pip install --target $PYTHON_SITE_PACKAGES pytest sre_yield requests pandas pyarrow findspark pytest-xdist pytest-order
+$PYSPARK_PYTHON -m pip install --target $PYTHON_SITE_PACKAGES -r ${SCRIPTPATH}/../../integration_tests/requirements.txt
+$PYSPARK_PYTHON -m pip install --target $PYTHON_SITE_PACKAGES requests pytest-order
```

Comment:
This does not look DBR-specific. Should we just move those over to `requirements` as well?

It is probably worth it to use requirements in this https://github.com/nartal1/spark-rapids/blob/a4e78d73ebb8b867b41930aac9dff03100a3d5e8/jenkins/Dockerfile-blossom.integration.ubuntu#L71 too cc @pxLi

Reply:
Can we solve that in a separate PR? This PR can then just solve this specific issue then.

Reply:
sure, cc @NvTimLiu to help later thanks